tests/test_game.py

```diff
@@ -85,6 +96,20 @@ def test_the_game_is_over_when_board_is_full():
 def test_play_turn_instructs_board_to_occupy_space():
     board = MockBoard()
     game = Game(board)
-    game.play_turn(space='1')
+    game.play_turn('1')
     assert board.occupy_space_call_count == 1
 
+
+def test_play_turn_raises_an_error_for_if_passed_an_occupied_space(game):
+    occupied_space = '8'
```

Comment:
Why do you define `occupied_space` inline but pass `invalid_space` in as a fixture?